doc/whats_new/v1.1.rst

```diff
@@ -69,6 +69,11 @@ Changelog
 - |Enhancement| All scikit-learn models now generate a more informative
   error message when setting invalid hyper-parameters with `set_params`.
   :pr:`21542` by :user:`Olivier Grisel <ogrisel>`.
+  
+- |Enhancement| All scikit-learn estimators now include the sklearn version
+  with which they have first been pickled when saving them with the pickle.
+  library.
```

Comment:
```suggestion
- |Enhancement| All scikit-learn estimators now include the sklearn version
  with which they have been pickled.
```

Reply:
I think it would be nice to make the pickled version public. There is a need if users are parsing the warning message for this information.